### DISCUSSION TITLE
Pairing some old Philips Hue bulbs 9290002269

### DISCUSSION THREAD

**Discussion by hasbean** [Koenkk/zigbee2mqtt#8907] [Devices]
Hello,

I finally got a ConBee II and it was immediately obvious that zigbee2mqtt is the way to go.  I got a bunch of stuff connected so far but for the life of me I can't get my old hue bulbs from 2014 to pair.

I noticed that it wasn't listed in the supported devices, but I thought at least I'd see the thing go through the pairing process.

The way I'm trying to reset is by pairing them with the Hue hub and then deleting the light.  Nothing shows up.  I search for them again on the Hue hub and they pair up there just fine.

I don't have the dimmer to do the reset trick but I don't think I needed it since I have the hub working just fine.

What did I miss exactly?  All my other stuff works great except these damn ancient bulbs

**Comment by hasbean** [comment]
Just an update: I bought a Philips Dimmer and gave that a go.  It worked without a hitch.

Nothing else worked for me other than the Philips Dimmer.  Reason seems to be it never sends an association request even if it's unpaired for some reason.

Ah well.

**Reply by MrApplejuice** [reply]
Hey, this sounds very promising - about the only thing I found how to get this to work!

Curious: What kind of dimmer did you get? There seem to be two kinds of dimmers on the market. I still have to of the old bulbs and would like to get them paired as well. Would be a shame to throw them out only because Philips does not feel like supporting them anymore.

**Reply by hasbean** [reply]
I got this one:  https://www.zigbee2mqtt.io/devices/324131092621.html

To be honest though, the integration was so finicky with zigbee2mqtt and things never really worked as expected, even after updating the bulbs.

 I just opted to use my old round Philips hue hub and added that to Home Assistant.  Not very ideal, but I guess it's a miracle these 9 year old bulbs are still working.

**Reply by MrApplejuice** [reply]
Hey! Thanks a lot for the info. I looked into it myself and given your comment and the fact that it is really hard to find that old-style dimmer nowadays, I did not try it out and donated the bulbs to a family member who still uses Philips Hue v1. I bought IKEA lamps instead. They work very well.
